src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -8027,4 +8029,103 @@ void wm_vuldel_truncate_revision(char * revision) {
     return;
 }
 
+int wm_vuldet_recv_wdb_by_row(cJSON** j_array) {
+    char* response = NULL;
+    char* payload = NULL;
+    const char* error = NULL;
+    int size;
```

Comment:
We should initialize size and parse_result.

Reply:
Done!

Reply:
Fixed